Header  header
float32 yawrate
float32 vx
float32 ax
float32 ay
float32 delta_swa
float32 load_axle_1
float32 load_axle_2
float32 load_axle_3
float32 speed_wheel_111
float32 speed_wheel_112
float32 speed_wheel_121
float32 speed_wheel_122
float32 speed_wheel_131
float32 speed_wheel_132
